proof/refine/RISCV64/Syscall_R.thy

```diff
@@ -1011,36 +1011,36 @@ crunches refillNew, refillUpdate, commitTime
   and ex_nonz_cap_to'[wp]: "ex_nonz_cap_to' ptr"
   (simp: crunch_simps wp: crunch_wps)
 
+lemma scSBadge_update_invs'[wp]:
+  "updateSchedContext scPtr (scBadge_update f) \<lbrace>invs'\<rbrace>"
+  apply (wpsimp wp: updateSchedContext_invs')
+  apply (intro conjI impI allI)
+    apply (erule (1) live_sc'_ko_ex_nonz_cap_to')
```

Comment:
Is this erule very sensitive to things not being clarsimped before? Or maybe it's not in simp-normal form? Otherwise it looks tempting to give everything after the wpsimp to a fastforce. (similar temptation for `scSporadic_update_invs'`

Reply:
I was able to compress this into one `fastforce` after the `wpsimp` in both proofs.